pdns/dnsdistdist/docs/install.rst

```diff
@@ -62,7 +62,7 @@ dnsdist depends on the following libraries:
 * `re2 <https://github.com/google/re2>`_ (optional)
 * `TinyCDB <https://www.corpit.ru/mjt/tinycdb.html>`_ (optional, CDB support)
 
-Since 2.0.0, the optional ``yaml`` configuration requires a Rust compiler and a Python 3 interpreter.
+Since 2.0.0, the optional ``yaml`` configuration requires a Rust compiler.
```

Comment:
and `cargo` ?

Reply:
Or maybe "a Rust development environment, including `rustc` and `cargo`?
Same below